test/text.js

```diff
@@ -63,3 +63,20 @@ test('Nodes', (t) => {
 
   t.end()
 })
+
```

Comment:
Below are some ideas on cleaning this code.
Optionally: it can also be dropped, as this test (and more) are now in `trim-lines`!

Reply:
I think I'll just drop this test, I'll make the change once the library has the final revision in place. See my PR for details. https://github.com/wooorm/trim-lines/pull/5

Reply:
sounds good